src/SceneKit/SCNCompat.cs

```diff
@@ -52,16 +53,32 @@ public virtual void SetTimingFunction (Action<float> timingFunction)
 	}
 #if TVOS && !XAMCORE_4_0
 	partial class SCNMaterialProperty {
+#if !NET
 	[iOS (8, 0)]
 		[Deprecated (PlatformName.iOS, 10, 0)]
 		[Deprecated (PlatformName.TvOS, 10, 0, message: "This API has been totally removed on tvOS.")]
+#else
+		[UnsupportedOSPlatform ("ios10.10")]
```

Comment:
No Obsolete attribute on iOS?

Reply:
Same with this selector!

Reply:
Looking at the headers, the same message applies.